CovertInfiltration/Src/CovertInfiltration/Classes/UIUtilities_Infiltration.uc

```diff
@@ -304,6 +306,11 @@ static function AssaultMissionAvaliable (XComGameState_MissionSite MissionState)
 	GeoscapeOpportunityAvaliable(MissionState, default.strAssaultAvaliableHeader, MissionState.GetUIButtonIcon(), MissionState.GetMissionObjectiveText());
 }
 
+static function AssaultMissionExpiring (XComGameState_MissionSite MissionState)
+{
+	GeoscapeOpportunityAvaliable(MissionState, default.strAssaultExpiringHeader, MissionState.GetUIButtonIcon(), MissionState.GetMissionObjectiveText(), eUIState_Warning);
```

Comment:
Not a fan of calling `GeoscapeOpportunityAvaliable` here as the name/purpose obviously clashes. If it's suitable, then perhaps rename it to `GeoscapeOpportunityNotification`?